packages/dev/loaders/src/STL/stlFileLoader.ts

```diff
@@ -159,15 +159,16 @@ export class STLFileLoader implements ISceneLoaderPlugin {
             return true;
         }
 
-        // check characters higher than ASCII to confirm binary
-        const fileLength = reader.byteLength;
-        for (let index = 0; index < fileLength; index++) {
-            if (reader.getUint8(index) > 127) {
+        // US-ASCII begin with 's', 'o', 'l', 'i', 'd'
+        const ascll = [ 115, 111, 108, 105, 100 ];
+        for ( let off = 0; off < 5; off ++ ) {

```

Comment:
Try to run prettier on the file, it will fix the formatting mistakes in the file (missing/additional blank spaces).